src/platform-includes/profiling/index/why-profiling/javascript.mdx

```diff
@@ -0,0 +1,5 @@
+## Why profiling?
+
+With [profiling](/product/profiling/), Sentry tracks your software's performance by sampling your program's call stack in a variety of environments, collecting function level information about your code and enables you to fine tune the performance of your program. Sentry profiling captures function calls and their exact locations, aggregates them and shows you the most common code paths of your program, which surfaces areas of optimization opportunity that can help you increase both the performance of your code and increase user satisfaction.
+
+In a browser environment, profiling can help you pinpoint causes of UI jank and surface why values like input to next paint are performing poorly or why a long task was blocking the browser from repainting the screen and causing frame drops. All of this information enables you to fix real world performance issues and deliver a snappier user experience to your users.
```

Comment:
```suggestion

In a browser environment, profiling can help you pinpoint causes of UI jank and surface why values like input to next paint are performing poorly or why a long task was blocking the browser from repainting the screen and causing frame drops. All of this information enables you to fix real world performance issues and deliver a snappier user experience to your users.

Because the profiling API is currently only exposed in Chromium, the profiles collected only include that demographic. We hope that as the API gains adoption, other browsers will implement it as well.
```
I would move this to live at the bottom of the intro section. I don't think it's important enough to be the first thing users read :)